Source/DrawableViewController.swift

```diff
@@ -48,6 +48,10 @@ class DrawableViewController: UIViewController {
     var saveImage: ((UIView) ->())?
     var didSelectShape: (() -> ())?
     private var currentShape: ShapeView?
+    private var shapeViewOffset: CGFloat = 50.0
+    
+    // property to store previous drawed image
+    weak var currentImageView: UIImageView!
```

Comment:
does this image always exists?

Reply:
Property deleted since we are not using it :)